recirq/qcqmc/hamiltonian.py

```diff
@@ -0,0 +1,319 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import attrs
+import fqe
+import h5py
+import numpy as np
+import openfermion as of
+from fqe.hamiltonians.restricted_hamiltonian import RestrictedHamiltonian
+from fqe.openfermion_utils import integrals_to_fqe_restricted
+from openfermionpyscf._run_pyscf import compute_integrals  # type: ignore
+from pyscf import ao2mo, fci, gto, scf
+
+from recirq.qcqmc.config import OUTDIRS
+from recirq.qcqmc.data import Data, Params, get_integrals_path
+
+
+@attrs.frozen
+class LoadFromFileHamiltonianParams(Params):
+    """Class for storing the parameters that specify loading integrals from a file."""
```

Comment:
Missing docstring about the vairables, such as n_orb, etc.

Reply:
Done.